### PR TITLE
[Warrior] Update *_Generate_Warrior.simc

### PR DESCRIPTION
**Pull request by Joromonni**

### CODE REVIEW

File: profiles/generators/PreRaids/PR_Generate_Warrior.simc [hunk 1]
```diff
@@ -1,79 +1,78 @@
...
-# trinket1=the_first_sigil,id=188271,bonus_id=1524/7187
-# trinket2=ticking_sack_of_terror,id=185844,bonus_id=1595/6536/6646
-# main_hand=devastated_worldshard,id=189851,bonus_id=7187/1524,enchant=celestial_guidance
-# off_hand=ruined_crest_of_lordaeron,id=189845,bonus_id=7187/1524
+head=waxsteel_greathelm,id=221100,ilevel=593
+neck=strapped_rescue_keg,id=221060,ilevel=593
+shoulders=flesh_connectors_epaulets,id=221185,ilevel=593
+back=anvilhide_cape,id=221088,ilevel=593
+chest=slashproof_business_plate,id=221069,ilevel=593,enchant=crystalline_radiance_2
+wrists=vociferous_subjugators_bracers,id=221167,ilevel=5938
```
Comment (Fluttershy-git) [inline profiles/generators/PreRaids/PR_Generate_Warrior.simc]: I'd like to have those Wrists, please

Comment (Joromonni) [reply profiles/generators/PreRaids/PR_Generate_Warrior.simc]: 😅 ah my bad. Should be fixed now

Comment (Joromonni) [reply profiles/generators/PreRaids/PR_Generate_Warrior.simc]: DMed w Archimtiros, fixing typos and a changing a few items.

**Review by Joromonni — COMMENTED** [review]
Everything should be in order now.

**Pull request merged**